subprojects/shipkit/src/test/groovy/org/shipkit/internal/comparison/artifact/BintrayDefaultArtifactUrlResolverTest.groovy

```diff
@@ -23,13 +31,15 @@ class BintrayDefaultArtifactUrlResolverTest extends Specification {
         result == "https://bintray.com/shipkit/examples/download_file?file_path=mockito/api/0.0.1/api-0.0.1-sources.jar"
     }
 
-    private BintrayExtension.PackageConfig mockBintrayPkg() {
-        def extensionContainer = Mock(ExtensionContainer)
-        project.getExtensions() >> extensionContainer
-        def extension = Mock(BintrayExtension)
-        extensionContainer.getByType(BintrayExtension.class) >> extension
-        def pkg = Mock(BintrayExtension.PackageConfig)
-        extension.pkg >> pkg
-        pkg
+    def "concatenates default url with fallback to user correctly"() {
+        given:
+        underTest = new BintrayDefaultArtifactUrlResolver(project, "api", "0.0.1")
+        project.getGroup() >> "mockito"
+        extension.user >> "shipkit"
```

Comment:
Let's make the value of this variable "shipkit-user", and in case of the other test pkg.userOrg >> "shipkit-org". It will make it more clear what we expect to happen. 

Also we share some of the setup between tests, which might lead to a situation when someone sets pkg.userOrg in setup method and then we would no longer use extension.user for this test but the test would still pass.